clientUtils/Util.ts

```diff
@@ -924,7 +924,10 @@ export const logMe = (
     return descriptor
 }
 
-export const splitArrayIntoGroupsOfN = (arr: any[], maxPerGroup: number) => {
+export const splitArrayIntoGroupsOfN = (
+    arr: any[],
+    maxPerGroup: number
+): any[] => {
```

Comment:
Can this one be easily parameterised too?

```suggestion
export const splitArrayIntoGroupsOfN = <T>(
    arr: T[],
    maxPerGroup: number
): T[][] => {
```

Reply:
Yes, nice! I'll do another round later of removing all any's but good to take this one down right now :)